src/theme/shared/global.css

```diff
@@ -1,11 +1,14 @@
 @import url('https://fonts.googleapis.com/css?family=Anton%7CMontserrat:500,700,800&display=swap');
 
 html {
+  -webkit-box-sizing: border-box;
   box-sizing: border-box;
 }
+
 *,
 *:before,
 *:after {
+  -webkit-box-sizing: inherit;
```

Comment:
do we need to add vender prefix?

Reply:
No really, it added automatically by the editor

Reply:
Removed